Cargo.toml

```diff
@@ -39,7 +40,7 @@ libdeflate = ["hts-sys/libdeflate"]
 s3 = ["hts-sys/s3"]
 gcs = ["hts-sys/gcs"]
 static = ["hts-sys/static"]
-serde = ["serde_base", "serde_bytes"]
+serde_features = ["serde_base", "serde_bytes"]
```

Comment:
If you revert the change on line 28, you be able to able revert this change.  There's no need to change the name of the feature. (`serde` is the standard name used in all crates that do this, and changing the feature name break backward compatibility).